### PR TITLE
Make the language consistent for describing ops.

### LINKED ISSUE #120: Improve descriptions of operations in list of operations
The descriptions for each of the operations are not parallel and consistent in how they are explained.
https://www.stellar.org/developers/learn/concepts/list-of-operations.html#path-payment

Examples of non-parallelism:
- First person: `Allows you to set,modify or delete a Data Entry (name/value pair) that is attached to a particular account.`
- Action: `Sends an amount in a specific asset to a destination account.`
- Overview description: `A passive offer is an offer that does not act on and take a reverse offer of equal price.`
- "this" description: `This operation sets the options for an account.`

### PR DESCRIPTION
**Pull request by theaeolianmachine**
Fixes #120.

### CODE REVIEW

File: guides/concepts/list-of-operations.md [hunk 1]
```diff
@@ -253,11 +256,16 @@ Possible errors:
 ## Set Options
 [JavaScript](http://stellar.github.io/js-stellar-sdk/Operation.html#.setOptions) | [Java](http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/SetOptionsOperation.Builder.html) | [Go](https://godoc.org/github.com/stellar/go/txnbuild#SetOptions)
 
-This operation sets the options for an account.
+Sets options for an account, such as setting the inflation destination or adding an additional
+signer on an account.
+
+This operations allows you to set multiple options on an account in a single operation, such as
```
Comment (ire-and-curses) [inline guides/concepts/list-of-operations.md]: "Allows multiple options on an account with a single operation" might fit the action ethic better here.

File: guides/concepts/list-of-operations.md [hunk 2]
```diff
@@ -253,11 +256,16 @@ Possible errors:
...
+changing a operation threshold and setting the flags on an account at the same time.
```
Comment (ire-and-curses) [inline guides/concepts/list-of-operations.md]: `a operation threshold` -> `an operation threshold`

File: guides/concepts/list-of-operations.md [hunk 3]
```diff
@@ -253,11 +256,16 @@ Possible errors:
...
 
-For more information on the signing options, please refer to the [multi-sig doc](./multi-sig.md).
+For more information on the options related to signing, please refer our documentation on
```
Comment (ire-and-curses) [inline guides/concepts/list-of-operations.md]: Could consider being terser: `For more information on the options related to signing, see [multi-sig](./multi-sig.md).`

File: guides/concepts/list-of-operations.md [hunk 4]
```diff
@@ -373,7 +381,11 @@ Possible errors:
 ## Inflation
 [JavaScript](http://stellar.github.io/js-stellar-sdk/Operation.html#.inflation) | [Java](http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/InflationOperation.html) | [Go](https://godoc.org/github.com/stellar/go/txnbuild#Inflation)
 
-This operation runs inflation.
+Run the inflation process for the entire Stellar network.
```
Comment (ire-and-curses) [inline guides/concepts/list-of-operations.md]: `Run` -> `Runs`

File: guides/concepts/list-of-operations.md [hunk 5]
```diff
@@ -373,7 +381,11 @@ Possible errors:
...
+
+Anyone can submit the Inflation operation that triggers the inflation process in the Stellar
```
Comment (ire-and-curses) [inline guides/concepts/list-of-operations.md]: `in the Stellar` -> `on the Stellar`

File: guides/concepts/list-of-operations.md [hunk 6]
```diff
@@ -388,9 +400,13 @@ Possible errors:
 ## Manage Data
 [JavaScript](http://stellar.github.io/js-stellar-sdk/Operation.html#.manageData) | [Java](http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/ManageDataOperation.Builder.html) | [Go](https://godoc.org/github.com/stellar/go/txnbuild#ManageData)
 
-Allows you to set,modify or delete a Data Entry (name/value pair) that is attached to a particular account. An account can have an arbitrary amount of DataEntries attached to it. Each DataEntry increases the minimum balance needed to be held by the account.
+Sets, modifies, or deletes a Data Entry (name/value pair) that is attached to a particular account.
```
Comment (ire-and-curses) [inline guides/concepts/list-of-operations.md]: `Data Entry` -> `data entry`

File: guides/concepts/list-of-operations.md [hunk 7]
```diff
@@ -388,9 +400,13 @@ Possible errors:
...
+
+An account can have an arbitrary amount of data entries attached to it. Each data entry increases
+the minimum balance (via the base reserve) needed to be held by the account.
 
-DataEntries can be used for application specific things. They are not used by the core Stellar protocol.
+Data entries can be used for storing application specific data on the Stellar Network. They are not
```
Comment (ire-and-curses) [inline guides/concepts/list-of-operations.md]: `application specific` -> `application-specific`

File: guides/concepts/list-of-operations.md [hunk 8]
```diff
@@ -388,9 +400,13 @@ Possible errors:
 ## Manage Data
 [JavaScript](http://stellar.github.io/js-stellar-sdk/Operation.html#.manageData) | [Java](http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/ManageDataOperation.Builder.html) | [Go](https://godoc.org/github.com/stellar/go/txnbuild#ManageData)
 
-Allows you to set,modify or delete a Data Entry (name/value pair) that is attached to a particular account. An account can have an arbitrary amount of DataEntries attached to it. Each DataEntry increases the minimum balance needed to be held by the account.
+Sets, modifies, or deletes a Data Entry (name/value pair) that is attached to a particular account.
+
+An account can have an arbitrary amount of data entries attached to it. Each data entry increases
```
Comment (ire-and-curses) [inline guides/concepts/list-of-operations.md]: "arbitrary" is not strictly true, although the number is large (1000 I think).

File: guides/concepts/list-of-operations.md [hunk 9]
```diff
@@ -413,9 +429,12 @@ Possible errors:
 ## Bump Sequence
 [JavaScript](http://stellar.github.io/js-stellar-sdk/Operation.html#.bumpSequence) | [Java](http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/BumpSequenceOperation.Builder.html) | [Go](https://godoc.org/github.com/stellar/go/txnbuild#BumpSequence)
 
-*Only available in protocol version 10 and above*
+*Only available in Stellar Protocol >= v10*
+
+Bump forward the sequence number of the source account to the given sequence number.
```
Comment (ire-and-curses) [inline guides/concepts/list-of-operations.md]: `Bump` -> `Bumps`

File: guides/concepts/list-of-operations.md [hunk 10]
```diff
@@ -413,9 +429,12 @@ Possible errors:
...
 
-Bump sequence allows to bump forward the sequence number of the source account of the operation, allowing to invalidate any transactions with a smaller sequence number.
+This operation allows you to invalidate any transactions with a smaller sequence number, and is
```
Comment (ire-and-curses) [inline guides/concepts/list-of-operations.md]: `This operation allows you to invalidate any` -> `This operation invalidates any`

File: guides/concepts/list-of-operations.md (hunk 2 above)
Comment (theaeolianmachine) [reply guides/concepts/list-of-operations.md]: good catch

File: guides/concepts/list-of-operations.md (hunk 8 above)
Comment (theaeolianmachine) [reply guides/concepts/list-of-operations.md]: An account can have a large amount of data entries attached to it (subject to sub-entry limits for
an account). Each data entry increases the minimum balance (via the base reserve) needed to be held
by the account.

**Comment by theaeolianmachine** [comment]
Went ahead and worked through all of those — left a comment for one of them, let me know if there are any new issues otherwise I'll merge this in later today.

**Pull request merged**